src/components/GUI/mainPage/navBar.tsx

```diff
@@ -0,0 +1,46 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import React from "react";
+import Nav from "react-bootstrap/Nav";
+import Navbar from "react-bootstrap/Navbar";
+import NavDropdown from "react-bootstrap/NavDropdown";
+import { Props } from "../../../App";
+import navBar from "../../CSS/navBar.module.scss";
+
+interface State {
+	name: string;
+	href: string;
+}
+
+export class NavBar extends React.Component<Props, State> {
```

Comment:
Nit: I'd use more expressive names, e.g. `NavigationBar`. It's easier to read and understand.